source/fundamentals/connection/connect.txt

```diff
@@ -65,11 +71,16 @@ to change the servers in rotation without reconfiguring clients.
    and select Java from the language dropdown to retrieve your connection
    string.
 
-The next part of the connection URI contains your credentials if you are
-using a password-based authentication mechanism. Replace the value of ``user``
-with your username and ``pass`` with your password. If your
-authentication mechanism does not require credentials, omit this part of
-the connection URI.
+If you are connecting to an instance or replica set that does not have a 
+DNS SRV address, you must use ``mongodb`` for the protocol, which specifies 
+the :manual:`Standard Connection String Format 
+</reference/connection-string/#std-label-connections-standard-connection-string-format>`.
+
+After the protocol, the next part of the connection string contains
```

Comment:
Suggestion: I think "after the protocol" and "the next part" mean the same thing. I think it could be simplified to:

```suggestion
After the protocol, the connection string contains your
```